src/CheckOut.js

```diff
@@ -755,6 +757,7 @@ class CheckOut extends React.Component {
           entityType="user"
           popUpPropertyName="popUpOnCheckOut"
           entityId={patron?.id}
+          label={intl.formatMessage({ id: 'ui-checkout.notes.popupModal.label' })}
```

Comment:
@Terala-Priyanka I see that you can use `FormattedMessage` directly without importing any additional wrappers like `injectIntl`.
```suggestion
          label={<FormattedMessage id="ui-checkout.notes.popupModal.label" />}
```

Reply:
@artem-blazhko 
Thank you for your comment. I have implemented accordingly.